pages/javascript-sdk/tutorials/run.mdx

```diff
@@ -358,6 +358,41 @@ $ npm run dev
 
 <ImageLoader alt="The Markdown Editor" img={previewUrl} />
 
+### (Optional) Automatically Compile the Rust
+
+It's possible to make `vite` automatically compile the `markdown-renderer` crate
+every time you run `npm run build` by using the [`onBuild`][onBuild] hook.
+
+```ts copy filename="vite.config.js"
+import { defineConfig } from "vite";
+import { exec } from "node:child_process";
+
+export default defineConfig({
+    ...
+    plugins: [
+        {
+            name: "cargo-build",
+            buildStart: () => {
+                return new Promise((resolve, reject) => {
+                    exec(
+                        "cargo build --target=wasm32-wasi --manifest-path=markdown-renderer/Cargo.toml --release --quiet",
```

Comment:
I'm not sure how cargo behaves if you pass `--manifest-path`, does it use `{provided_path}/target`, or does it use `./target`?

This could mess up the path.

Reply:
The `target/` directory is always next to your workspace's `Cargo.toml` file. The directory you are ein when you invoke `cargo build` doesn't have any effect on things.